compiler/qsc_parse/src/item/tests.rs

```diff
@@ -1693,3 +1682,37 @@ fn namespace_with_conflicting_names() {
                         Type _id_ [87-90]: Path: Path _id_ [87-90] (Ident _id_ [87-90] "Int")"#]],
     );
 }
+
+// We technically broke this syntax. Although we don't think anybody was using it,
```

Comment:
Can you be more specific as to when, e.g. "as of v1.3.1" or "May 2024".

This may be outdated thinking, but I've just read one too many comments like "we still need to support this for another few months!" and I had no way of knowing if that's from 2022 or 1996